environemnt set up
Running the batch script
2020-12-19 12:16:39.194183: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:16:53.932447: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-12-19 12:16:53.957460: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2900000000 Hz
2020-12-19 12:16:53.961259: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55ebe0638a90 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-12-19 12:16:53.961288: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-12-19 12:16:53.966489: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-12-19 12:16:54.196288: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55ebe063b290 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-12-19 12:16:54.196359: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Tesla V100S-PCIE-32GB, Compute Capability 7.0
2020-12-19 12:16:54.204885: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:3b:00.0 name: Tesla V100S-PCIE-32GB computeCapability: 7.0
coreClock: 1.597GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 1.03TiB/s
2020-12-19 12:16:54.204965: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:16:54.211242: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:16:54.216319: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:16:54.218303: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:16:54.224206: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:16:54.228321: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:16:54.238778: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:16:54.247508: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:16:54.249371: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:16:56.559303: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:16:56.559361: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:16:56.559374: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:16:56.563724: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100S-PCIE-32GB, pci bus id: 0000:3b:00.0, compute capability: 7.0)
2020-12-19 12:16:56.569013: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:3b:00.0 name: Tesla V100S-PCIE-32GB computeCapability: 7.0
coreClock: 1.597GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 1.03TiB/s
2020-12-19 12:16:56.569057: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:16:56.569083: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:16:56.569102: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:16:56.569118: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:16:56.569133: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:16:56.569147: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:16:56.569162: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:16:56.573030: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:16:56.573063: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:16:56.573071: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:16:56.573079: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:16:56.576886: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100S-PCIE-32GB, pci bus id: 0000:3b:00.0, compute capability: 7.0)
2020-12-19 12:16:57.038523: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:3b:00.0 name: Tesla V100S-PCIE-32GB computeCapability: 7.0
coreClock: 1.597GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 1.03TiB/s
2020-12-19 12:16:57.038580: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:16:57.038619: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:16:57.038631: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:16:57.038641: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:16:57.038658: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:16:57.038668: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:16:57.038678: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:16:57.041262: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:16:57.043507: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:3b:00.0 name: Tesla V100S-PCIE-32GB computeCapability: 7.0
coreClock: 1.597GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 1.03TiB/s
2020-12-19 12:16:57.043534: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:16:57.043549: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:16:57.043559: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:16:57.043568: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:16:57.043577: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:16:57.043585: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:16:57.043595: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:16:57.047034: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:16:57.047076: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:16:57.047083: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:16:57.047089: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:16:57.050534: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100S-PCIE-32GB, pci bus id: 0000:3b:00.0, compute capability: 7.0)
Using TensorFlow backend.
WARNING:tensorflow:AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x2b2d50b0c488> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
Does the system is built with CUDA?: True
Default GPU Device: /device:GPU:0
====> Start running baseline.py
Classes: ['Tomato___Late_blight', 'Tomato___Tomato_Yellow_Leaf_Curl_Virus', 'Peach___healthy', 'Grape___Esca_(Black_Measles)', 'Potato___Late_blight', 'Pepper,_bell___Bacterial_spot', 'Strawberry___healthy', 'Orange___Haunglongbing_(Citrus_greening)', 'Tomato___Leaf_Mold', 'Apple___Black_rot', 'Strawberry___Leaf_scorch', 'Tomato___Early_blight', 'Cherry_(including_sour)___healthy', 'Corn_(maize)___Common_rust_', 'Blueberry___healthy', 'Potato___Early_blight', 'Pepper,_bell___healthy', 'Apple___Cedar_apple_rust', 'Grape___Leaf_blight_(Isariopsis_Leaf_Spot)', 'Tomato___Tomato_mosaic_virus', 'Tomato___Target_Spot', 'Tomato___healthy', 'Peach___Bacterial_spot', 'Corn_(maize)___Cercospora_leaf_spot Gray_leaf_spot', 'Tomato___Spider_mites Two-spotted_spider_mite', 'Corn_(maize)___healthy', 'Squash___Powdery_mildew', 'Cherry_(including_sour)___Powdery_mildew', 'Tomato___Bacterial_spot', 'Grape___Black_rot', 'Apple___healthy', 'Potato___healthy', 'Corn_(maize)___Northern_Leaf_Blight', 'Tomato___Septoria_leaf_spot', 'Raspberry___healthy', 'Soybean___healthy', 'Apple___Apple_scab', 'Grape___healthy']
Number of classes: 38
Total number of images: 54305
first_folder_path: /fs/scratch/PAA0023/dong760/PlantVillage-Dataset/raw/color/Tomato___Late_blight
Number of folder: 38
sample image path: /fs/scratch/PAA0023/dong760/PlantVillage-Dataset/raw/color/Tomato___Late_blight/5fdf3288-d5f4-4c70-9b12-95222b82d415___RS_Late.B 4942.JPG
Image size: (256, 256, 3)
Found 43456 images belonging to 38 classes.
Found 10849 images belonging to 38 classes.
Train size: 43456

====> Statistics: epochs=5, batch_size=64, validation_split=0.2, lr=0.001, momentum=0.9, steps_per_epoch =100, feature shape= (256, 256, 3), no_classes=38, loss_function=categorical_crossentropy
Epoch 1/5
2020-12-19 12:17:09.253531: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:17:10.436948: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
WARNING:tensorflow:AutoGraph could not transform <function Model.make_test_function.<locals>.test_function at 0x2b2d51ff9268> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
679/679 - 1159s - loss: 0.5526 - accuracy: 0.8594 - precision: 0.9558 - recall: 0.7712 - auc: 0.9939 - val_loss: 0.2031 - val_accuracy: 0.9489 - val_precision: 0.9723 - val_recall: 0.9172 - val_auc: 0.9995
Epoch 2/5
679/679 - 1097s - loss: 0.1243 - accuracy: 0.9723 - precision: 0.9839 - recall: 0.9561 - auc: 0.9997 - val_loss: 0.1475 - val_accuracy: 0.9602 - val_precision: 0.9741 - val_recall: 0.9448 - val_auc: 0.9995
Epoch 3/5
679/679 - 1091s - loss: 0.0687 - accuracy: 0.9884 - precision: 0.9925 - recall: 0.9816 - auc: 0.9999 - val_loss: 0.1136 - val_accuracy: 0.9691 - val_precision: 0.9783 - val_recall: 0.9578 - val_auc: 0.9997
Epoch 4/5
679/679 - 1081s - loss: 0.0445 - accuracy: 0.9946 - precision: 0.9966 - recall: 0.9916 - auc: 0.9999 - val_loss: 0.1039 - val_accuracy: 0.9694 - val_precision: 0.9778 - val_recall: 0.9600 - val_auc: 0.9998
Epoch 5/5
679/679 - 1075s - loss: 0.0312 - accuracy: 0.9978 - precision: 0.9986 - recall: 0.9963 - auc: 1.0000 - val_loss: 0.0965 - val_accuracy: 0.9713 - val_precision: 0.9784 - val_recall: 0.9638 - val_auc: 0.9996
WARNING:tensorflow:From /users/PAA0023/dong760/miniconda3/envs/tf_latest/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2020-12-19 13:49:06.695829: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /users/PAA0023/dong760/miniconda3/envs/tf_latest/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
WARNING:tensorflow:AutoGraph could not transform <function canonicalize_signatures.<locals>.signature_wrapper at 0x2b2dd0918bf8> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
{'loss': [0.5526209473609924, 0.1243208795785904, 0.06871619075536728, 0.044536951929330826, 0.031152037903666496], 'accuracy': [0.859375, 0.9723398089408875, 0.9883560538291931, 0.9945922493934631, 0.997813880443573], 'precision': [0.9557678699493408, 0.983850359916687, 0.9924846887588501, 0.9966462850570679, 0.9985699653625488], 'recall': [0.771216869354248, 0.9560934901237488, 0.9815905690193176, 0.9916006922721863, 0.996272087097168], 'auc': [0.9938539862632751, 0.9996689558029175, 0.9999133348464966, 0.9999454617500305, 0.9999857544898987], 'val_loss': [0.2030743658542633, 0.14749307930469513, 0.11355739086866379, 0.10385482013225555, 0.09647905826568604], 'val_accuracy': [0.9489353895187378, 0.9601806402206421, 0.9691215753555298, 0.9693980813026428, 0.9713337421417236], 'val_precision': [0.9722520709037781, 0.9740568399429321, 0.9782527089118958, 0.9778424501419067, 0.9783849716186523], 'val_recall': [0.9172273874282837, 0.9447875618934631, 0.957784116268158, 0.9599963426589966, 0.9637754559516907], 'val_auc': [0.9994788765907288, 0.9994776248931885, 0.9996723532676697, 0.9997842311859131, 0.9995623230934143]}
date and time = 19-12-2020_13-48-59
Saving the model to path: /users/PAA0023/dong760/plant_leaves_diagnosis/saved_models/InceptionV3_BatchSize_64_0.2ValSplit_19-12-2020_13-48-59


p0309.ten.osc.edu:
                                                                               Req'd  Req'd   Elap 
Job id               Username Queue    Name                 SessID NDS   TSK   Memory Time Use S Time 
-------------------- -------- -------- -------------------- ------ ----- ----- ------ ----- - -----
2564371              dong760  gpuseria ondemand/sys/dashboa --         1    28     -- 05:00 R 03:59
2565109              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 C 01:32
2565110              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 R 01:32
2565113              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 R 01:32
2565115              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 R 01:32
2565116              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 R 01:30
2565118              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 R 01:30
2565119              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 R 01:30
The date when running current script is :
Sat Dec 19 13:49:29 EST 2020
